apps/test/unit/code-studio/pd/form_components_func/FormControllerTest.js

```diff
@@ -20,594 +21,567 @@ let DummyPage3 = () => {
+        setupErrored();
 
-        const setupErrored = (applicationId = undefined) => {
-          form = isolateComponent(
-            <FormController {...defaultProps} requiredFields={['field1']} />
-          );
-          DummyPage3.associatedFields = ['field1'];
-          setPage(2);
-        };
+        triggerSubmit();
 
-        beforeEach(() => {
-          server = sinon.fakeServer.create();
-        });
-        afterEach(() => {
-          server.restore();
-        });
+        expect(getErrors(DummyPage3)).to.not.be.empty;
+        expect(server.requests).to.be.empty;
+      });
 
-        it('Does not submit when the last page has errors', () => {
-          setupErrored();
+      [
+        {previouslySaved: false, message: 'with new application'},
+        {previouslySaved: true, message: 'with previously-saved application'}
+      ].forEach(({previouslySaved, message}) => {
+        it(`Submits when the last page has no errors ${message}`, () => {
+          previouslySaved
+            ? setUpValidWithApplicationId(applicationId)
+            : setupValid();
 
           triggerSubmit();
 
-          expect(getErrors(DummyPage3)).to.not.be.empty;
-          expect(server.requests).to.be.empty;
-        });
-
-        [
-          {previouslySaved: false, message: 'with new application'},
-          {previouslySaved: true, message: 'with previously-saved application'}
-        ].forEach(({previouslySaved, message}) => {
-          it(`Submits when the last page has no errors ${message}`, () => {
-            previouslySaved
-              ? setUpValidWithApplicationId(applicationId)
-              : setupValid();
-
-            triggerSubmit();
-
-            expect(getErrors(DummyPage3)).to.be.empty;
-            expect(server.requests).to.have.length(1);
-            expect(server.requests[0].method).to.eql(
-              previouslySaved ? 'PUT' : 'POST'
-            );
-            expect(server.requests[0].url).to.eql(
-              previouslySaved
-                ? `${fakeEndpoint}/${applicationId}`
-                : fakeEndpoint
-            );
-          });
+          expect(getErrors(DummyPage3)).to.be.empty;
+          expect(server.requests).to.have.length(1);
+          expect(server.requests[0].method).to.eql(
+            previouslySaved ? 'PUT' : 'POST'
+          );
+          expect(server.requests[0].url).to.eql(
+            previouslySaved ? `${fakeEndpoint}/${applicationId}` : fakeEndpoint
+          );
         });
+      });
 
-        it('Disables the submit button during submit and renders spinner', () => {
-          setupValid();
-          triggerSubmit();
-          expect(form.findOne('#submit').props.disabled).to.be.true;
-          expect(form.findAll('Spinner')).to.have.length(1);
-        });
+      it('Disables the submit button during submit and renders spinner', () => {
+        setupValid();
+        triggerSubmit();
+        expect(form.findOne('#submit').props).to.be.disabled;
```

Comment:
Updated from `expect(... .props.disabled).to.be.true` to `expect(... .props).to.be.disabled` for clearer failure messages